examples/platforms/nrf528xx/src/radio.c

```diff
@@ -107,8 +108,10 @@ static otInstance *  sInstance = NULL;
 static otRadioFrame sAckFrame;
 static bool         sAckedWithFramePending;
 
+static int8_t sMaxTxPowerTable[OT_RADIO_2P4GHZ_OQPSK_CHANNEL_MAX - OT_RADIO_2P4GHZ_OQPSK_CHANNEL_MIN + 1];
 static int8_t sDefaultTxPower;
```

Comment:
Should we set initialize value to the variable `sDefaultTxPower`?  The function `otPlatRadioReceive ()` may be called before the variable `sDefaultTxPower` is initialized.

Reply:
The behavior before this change is passed is to use 0 as the implicit default value (all static variables will be zero initialized). I prefer not to change this behavior when the max power table is not passed.